docs/source/releases/v1_14_0a0/documentation-better-release-notes.yaml

```diff
@@ -0,0 +1,5 @@
+documentation:
+- description: 'Added summaries to the minor releases in the docs to make searching for a specific change easier.'
+  title: 'Add top level descriptions to minor releases'
+  files:
+    - modified: "docs/source/releases/index.rst"
```

Comment:
Will these work now?  Are we ready to start doing release notes this way?  @jsolbrig if so, we'll need to get the CI running, and update template_basic_plugin repo accordingly ?

Reply:
Pending @jsolbrig determination if we are ready to get this CI set up right away